### PR TITLE
Create account

### PR DESCRIPTION
**Pull request by nathancohen4299**

### CODE REVIEW

File: depend_on_me_api/objs/User.py [hunk 1]
```diff
@@ -8,21 +8,24 @@ class User(UserMixin):
...
     def __init__(
         self,
         id: str = None,
         password: str = None,
         email: str = None,
         first_name: str = None,
         last_name: str = None,
-        privilege: int = None,
+        privilege: int = 0,
+        tags: list = []
```
Comment (fatcat2) [inline depend_on_me_api/objs/User.py]: For lists, uses `List`

Needs an import up above like the following:
```
from Typing import List
```

https://docs.python.org/3/library/typing.html

File: depend_on_me_api/routes/routes.py [hunk 2]
```diff
@@ -1,32 +1,40 @@
...
     template_folder="../../website/static/public/",
 )
 login_manager = LoginManager()
+db = firestore.Client()
 
 
 @backend.route("/", methods=["GET"])
 def default():
-    return "200 OK"
+    return "", status.HTTP_200_OK
```
Comment (fatcat2) [inline depend_on_me_api/routes/routes.py]: Look into `jsonify`

https://flask.palletsprojects.com/en/1.1.x/api/#flask.json.jsonify

Comment (stephend017) [reply depend_on_me_api/routes/routes.py]: you can also use a dict to store the result, they work well with the json library in  python

File: depend_on_me_api.egg-info/requires.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+flask
```
Comment (fatcat2) [inline depend_on_me_api.egg-info/requires.txt]: this should be called requirements.txt

File: depend_on_me_api/objs/User.py [hunk 4]
```diff
@@ -8,21 +8,25 @@ class User(UserMixin):
...
     def __init__(
         self,
         id: str = None,
         password: str = None,
         email: str = None,
         first_name: str = None,
         last_name: str = None,
-        privilege: int = None,
+        privilege: int = 0,
+        tags: list = [],
```
Comment (fatcat2) [inline depend_on_me_api/objs/User.py]: I think the correct way to type lists is like `List[type]`

File: start.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export GOOGLE_APPLICATION_CREDENTIALS="/Users/nathancohen/CohenWorkspace/Authentications/depend-on-me-c33aad1c6e15.json"
```
Comment (fatcat2) [inline start.txt]: What's this file doing here?

**Pull request merged**